### PR TITLE
Remove Participate from unauthenticated routes and rename Emission Estimation to Travel Planner

### LINKED ISSUE #191: rename "Emission Estimation" to "Travel Planner" in the header
unless it is not clear, which emissions are estimated here

### LINKED ISSUE #202: 404 for participate page
The http://localhost:3000/participate page is not found. Error: 404.

### PR DESCRIPTION
**Pull request by sami1riaz**

### CODE REVIEW

File: frontend/src/data/routes.tsx [hunk 1]
```diff
@@ -17,11 +17,9 @@ export const prodRoutes: Route[] = [
 // TODO: add translation
 export const routes: Route[] = [
   { name: "Demo", link: "/dashboard" },
-  { name: "Emission Estimation", link: "/emission-estimation"},
```
Comment (sami1riaz) [inline frontend/src/data/routes.tsx:20]: I felt this should be accessible only when logged in as well but lmk if my assumption was incorrect

File: frontend/src/factories/ChartDataFactory.tsx [hunk 2]
```diff
@@ -38,7 +38,6 @@ export function mapChartData(rawData: any, year: string){
         })
         return chartData;
     }
-}
```
Comment (sami1riaz) [inline frontend/src/factories/ChartDataFactory.tsx:41]: This was causing dashboard load to fail

**Comment by sami1riaz** [comment]
Should also close #191

File: frontend/src/factories/ChartDataFactory.tsx (hunk 2 above)
Comment (codingfabi) [reply frontend/src/factories/ChartDataFactory.tsx:41]: Good catch :)

File: frontend/src/data/routes.tsx (hunk 1 above)
Comment (codingfabi) [reply frontend/src/data/routes.tsx:20]: No these should be able to be used without authentication. They are explicitly designed to function without it. Pleaser revert :)

**Review by codingfabi — COMMENTED** [review]
Address requested changes please. Rest looks good :)

File: frontend/src/data/routes.tsx (hunk 1 above)
Comment (sami1riaz) [reply frontend/src/data/routes.tsx:20]: done

**Pull request merged**